### DISCUSSION TITLE
I have 2 Vercel projects, one is hosted on mydomain.com, how can I host the other one on mysubdomain.mydomain.com?

### DISCUSSION THREAD

**Discussion by t0t07** [vercel/vercel#5751] [Help] [answered]
I have 2 Vercel projects A and B, and I own a domain mydomain.com.

A is deployed to mydomain.com. And I want to deploy B to mysubdomain.mydomain.com.

I bet I need to add a DNS record for my Deployment B, but I am not sure what DNS record to add.

Any help? Thanks.

**Comment by jacobhq** [comment, marked as answer]
Just add a domain in your vercel project for deployment b as `yoursubdomain.yourdomain.com`. Then it will give you a DNS record for you to add at your registrar.

**Reply by t0t07** [reply]
Thank you! It works. Vercel is slick
